doc/installation/system/inifile.rst

```diff
@@ -155,4 +155,37 @@ can give these different nodes, dedicated names.
 all available nodes in the cluster.
 
 If ``PI_NODE`` is not set, then ``PI_AUDIT_SERVERNAME`` is used as node name.
-If this is also not set, the node name is returned as "localnode".
\ No newline at end of file
+If this is also not set, the node name is returned as "localnode".
+
+.. _trusted_jwt:
+
+Trusted JWTs
+-------------
+
+Other applications can use the API without the need
+to call the ``/auth`` endpoint. This can be achieved by
+trusting private RSA keys to sign JWTs. You can define a list
+of corresponding public keys that are trusted for certain
+users and roles using the parameter ``PI_TRUSTED_JWT``::
+
+   PI_TRUSTED_JWT = [{"public_key": "-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEF...",
+                       "algorithm": "RS256",
+                       "role": "user",
+                       "realm": "realm1",
+                       "user": "userA",
+                       "resolver": "resolverX"}]
+
+
+This entry means, that the private key, that corresponds the the given
+public key can sign a JWT, that can impersonate as the *userA* in resolver
+*resolverX* in *realmA*.
+
+A JWT can be signed like this::
```

Comment:
created?

Reply:
I think I really ment *signed*, since the ``dict`` is somehow signed with the key. 
But you are right, *created* sounds more sensible.

Will do.

Reply:
done in 5ce20321